metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/force_delete_test.clj

```diff
@@ -66,3 +75,82 @@
        ;; tag association associated with other collection revision or whole collection is not deleted
        (util/is-tag-association-deleted? ta1 false)
        (util/is-tag-association-deleted? ta3 false)))))
+
+(deftest force-delete-service
```

Comment:
I think we should add one more test. Force delete the latest revision which is a tombstone. It shouldn't impact any associations, just want to make sure that it successfully removes the tombstone, and that a call to retrieve the service now finds it.

Reply:
This is kind of unnecessary as we should not have any associations when the latest revision is a tombstone, but we can add a removal of a tombstone revision just for fun.

Reply:
Just following up.

The point of the test is not for associations, it's just a test to make sure force delete on tombstones works as expected (the record gets deleted and the service is now found by a search). This actually makes more sense for an integration test with search (so I'm fine with not adding it to this namespace) to make sure that the service is now indexed (it wasn't before when the latest revision was a tombstone). The last I checked I didn't see code that actually made this work.